src/devrules/core/git_service.py

```
@@ -428,3 +430,17 @@ def get_author() -> str:
         return result.stdout.strip()
     except subprocess.CalledProcessError:
         return "Unknown Author"
+
+
+def get_current_repo_name() -> str:
+    """Get the current git repository name."""
+    try:
+        result = subprocess.run(
+            ["git", "config", "--get", "remote.origin.url"],
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)